examples/auth/token/main.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2023-2024 Deepgram SDK contributors. All Rights Reserved.
+# Use of this source code is governed by a MIT license that can be found in the LICENSE file.
+# SPDX-License-Identifier: MIT
+
+import os
+from dotenv import load_dotenv
+import logging
+from deepgram.utils import verboselogs
+
+from deepgram import (
+    DeepgramClient,
+    PrerecordedOptions,
+    DeepgramClientOptions
+)
+
+load_dotenv()
+
+AUDIO_URL = {
```

Comment:
Is this being used in this example? All I got back was the token response.

Reply:
Nope, thanks for catching!